### ISSUE TITLE
[Feature Request] differentiate domains by port 

### ISSUE THREAD

**Issue by Bertrand** [ProxymanApp/Proxyman#1049]
When developing several http services at once, most services are served by localhost, each on its own port. 

In that case, there is no way to pin only the traffic on one port, and all requests end-up in the same list: 

<img width="855" alt="Screenshot 2021-10-27 at 09 10 45" src="https://user-images.githubusercontent.com/4960/139017641-c1f25ca8-2a3d-4f46-b6af-10a96c24e073.png">

When the list contains dozen of intertwined requests, it's very hard to follow the sequence of requests on a particular service.

It would be great to be able to edit a "pinned" entry and add a port to only see the traffic related to this port. 
Of course, we should be able to pin the same domain twice, with a different port. 

As for the "Domains" view, Charles Proxy includes the port in domain in Structure View to address this particular point: 

<img width="432" alt="image" src="https://user-images.githubusercontent.com/4960/139016589-76f92281-c677-4719-b45d-ef3aa8eb2234.png">

This behaviour could be controlled by a global setting.

**Comment by Bertrand** [comment]
Hello @NghiaTranUIT. 

I've been running on this version for a few days now. No regression found for the moment 👍 
And having the traffic well-separated by service is an awesome improvement. Thanks :clap: :clap:

**Comment by NghiaTranUIT** [comment]
Awesome, glad to hear that 😄 🙌
